src/Main.java

```diff
@@ -0,0 +1,98 @@
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Main {
+
+    static final String ANSI_GREEN = "\u001B[32m";
+    static final String ANSI_RESET = "\u001B[0m";
+    static final String RED_BOLD = "\033[1;31m";
+
+    static CheckInput check = new CheckInput();
+    static LogIn log = new LogIn();
+    static Scanner scanner = new Scanner(System.in);
+    static Services services = new Services();
+
+    public static void main(String[] args) throws IOException {
+
+        final String logInFile = "logInFile.txt";
+        String userName, PIN;
+        String option;
+        int counter = 1;
```

Comment:
try to be more specific, what this count?

Reply:
renamed to numberOfLoginTrials.